queries/highlights.scm

```diff
@@ -0,0 +1,27 @@
+[
```

Comment:
I'm guessing this is specific to nvim, correct? Could it be moved to a different library, like [this one](https://github.com/victorhqc/language-prisma)?

Reply:
If it's something more or less standard, then I'm happy to merge it. Just want to avoid marrying this library to `nvim`. Just explore if it's possible to have it in a diff library 😄

Reply:
Hey, good question. I looked around to other parser repos and they all follow this approach however couldn't find concrete documentation about it. I believe it has to do with the fact that `nvim-treesitter` offers a `TSInstallFromGrammar` that uses the `grammar.js` on the repo so if I had to guess that's why they tend to live here.

Reply:
Ah, gotcha 🤷‍♂️ thanks for checking. Then I'm happy to add it to the repo. Thanks for adding it!